api/DHSC.FingertipsNext.Modules.Core/LoggingExtensions.cs

```diff
@@ -0,0 +1,7 @@
+namespace DHSC.FingertipsNext.Modules.Core;
+
+public static partial class LoggingExtensions
+{
+    [LoggerMessage(0, LogLevel.Error, "Log Example failed operation during: {Context}")]
```

Comment:
Should both log examples have an event ID of 0?

Reply:
Fair - I've updated one of them to show a different event ID